runtime/doc/api.txt

```diff
@@ -1711,6 +1711,23 @@ nvim_call_function({fn}, {args})                        *nvim_call_function()*
                 Return: ~
                     Result of the function call
 
+nvim_cmd({*cmd}, {*opts})                                         *nvim_cmd()*
```

Comment:
It's confusing to have two functions in the API called `nvim_cmd` and `nvim_command` that do very similar things. Should we at least say in the docs a little bit about why there are two functions for this and summarize the difference between them or when you should prefer one over the other?

Reply:
That's the way the API goes: functions that are added in a release cannot be removed (before 2.0). So some duplication is inevitable (and has happened before).

Reply:
That all sounds good, but then should we say a bit in the docs about why there are two functions and when people should prefer to use which?

Reply:
Resolved, TYVM!